ui/packages/catalog-ui-search/src/main/webapp/component/search-form/forms-sharing/search-form-sharing-tab-container.view.js

```diff
@@ -27,26 +28,32 @@ module.exports = Marionette.LayoutView.extend({
     collectionView: '.collection',
   },
   initialize: function() {
-    this.searchFormSharingCollection = SearchFormsSharingCollection
+    this.searchFormCollection = SearchFormCollection
     this.listenTo(
-      this.searchFormSharingCollection,
+      this.searchFormCollection,
       'change:doneLoading',
       this.handleLoadingSpinner
     )
   },
   onRender: function() {
     this.collectionView.show(
       new SearchFormCollectionView({
-        collection: this.searchFormSharingCollection.getCollection(),
+        collection: this.searchFormCollection.getCollection(),
         model: this.model,
-        hideInteractionMenu: this.options.hideInteractionMenu,
+        filter: function(child) {
+          return (
+            child.get('createdBy') !== user.getEmail() &&
+            user.canRead(child) &&
```

Comment:
Based on `user.canRead(child)`, do we send back items that the user shouldn't be able to see?

Reply:
I believe so but @rymach would know more. Are you thinking they should be filtered on the backend instead?

Reply:
they should be filtered on the backend, but when I was working this that wasn't always the case. @Schachte